src/components/Form/Form.js

```diff
@@ -1280,23 +1281,25 @@ class Form extends Component {
     }
 
     renderCourseRegistrationSubmission(){
+        console.log(this.state)
         if(this.props.registeredCourses){
 
             let currentStudentID = this.state.Student.Student.value;
             let registeredCourseForm = this.props.registeredCourses[currentStudentID];
             registeredCourseForm = registeredCourseForm[registeredCourseForm.length - 1];
-
             let currentStudentName = registeredCourseForm.display.student_name;
             let currentCourseTitle = registeredCourseForm.display.course_name;
+            let student=currentStudentName.slice(0,currentStudentName.indexOf(" -"));
+            let email=currentStudentName.slice(currentStudentName.indexOf("- "), currentStudentName.length);
+            let instructorName=this.props.instructors[registeredCourseForm.new_course.instructor].name;
+            let studentID=registeredCourseForm.student_id;
+            let courseName=currentCourseTitle.slice(currentCourseTitle.indexOf("- "), currentCourseTitle.length);
+            let startDate=registeredCourseForm.new_course.schedule.start_date;
+            console.log(instructorName);
 
-
+            console.log(registeredCourseForm);
             return <div>
-                <h3>{currentStudentName}</h3>
-                <h3>{currentCourseTitle}</h3>
-                <Button component={NavLink} to={"/registration"}
-                        className={"button"}>Register More</Button>
-                <Button component={NavLink} to={"/registration/cart"}
-                        className={"button"}>Checkout</Button>
+                <CompleteCourseRegistration startDate={startDate} student={student} studentID={studentID} courseName={courseName} email={email} instructorName={instructorName} courseName={courseName} currentStudentName={currentStudentName} currentCourseTitle={currentCourseTitle}/>
```

Comment:
1. Rename the component to "CourseRegistrationReceipt" 

2. Don't give so many props. gather everything together into an object. 

example: 
```
const registrationInfo = {
       startDate: startDate,
       student: student,
       ...etc,
}
```
Then put it into the component as 1 prop.

example `<CompleteCourseRegistration registrationInfo = {registrationInfo}/>` 

If you need to give so many props, be sure to give each prop a new line. this is industry standard

example: 
```
<CompleteCourseRegistration 
                  startDate={startDate}
                  student={student}
                  etc....
/>
```